src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtil.java

```diff
@@ -181,6 +181,29 @@ public static boolean isInInterfaceBlock(DetailAST node) {
         return isInBlockOf(node, TokenTypes.INTERFACE_DEF);
     }
 
+    /**
+     * Returns whether a node is directly contained within an anonymous object block.
+     *
+     * @param node the node to check if directly contained within an anonymous object block.
+     * @return a {@code boolean} value
+     */
+    public static boolean isInObjBlock(DetailAST node) {
```

Comment:
Let's keep this as a private method in `ClassMemberImpliedModifier` until we can use it elsewhere.

Reply:
Done